other/docker/coverage/coverage.Dockerfile.dockerignore

```diff
@@ -0,0 +1,24 @@
+# ===== common =====
+# Ignore everything ...
+**/*
+# ... except sources
+!**/*.[ch]
+!**/*.cc
+!**/*.hh
+!CHANGELOG.md
+!LICENSE
+!README.md
+!auto_tests/data/*
+!other/bootstrap_daemon/bash-completion/**
+!other/bootstrap_daemon/tox-bootstrapd.*
+!other/proxy/*.mod
+!other/proxy/*.sum
+!other/proxy/*.go
+# ... and CMake build files (used by most builds).
+!**/CMakeLists.txt
+!.github/scripts/flags*.sh
+!cmake/*.cmake
+!other/pkgconfig/*
+!other/rpm/*
+!so.version
+!other/docker/coverage/run_mallocfail
```

Comment:
Was this file committed by a mistake? It's supposedly auto-generated by `../sources/run.sh` when called by `coverage/run`.

Reply:
Unfortunately, it's not a mistake. This file has to be in git because otherwise a default "build this docker file" action (github or render.com) doesn't work the way it should.

Reply:
Huh. So, whenever someone changes the master dockerignore file in either `other/docker/sources/`, or a supplemental one in `other/docker/$BUILD/`, they would have to run the `other/$BUILD/run` script(s) to update the concatenated dockerignore file(s) before committing/making a PR?

Maybe we could add a script step before the "build this docker file" action that would generate this dockerignore file? For example, move the dockerignore concat section out of `other/docker/sources/ru.sh` into its own script, something like `other/docker/source/gen-dockerignore.sh` that takes `$BUILD` as an argument (or a path to `$BUILD`'s directory or something like that) and concats the master dockerignore with the `$BUILD`'s, putting it in `$BUILD`'s directory? That script could be used in CI to generate the dockerignore file before calling the docker actions and will also be called from `other/docker/sources/run.sh`.

(Also, what is this render.com promotion? How is that relevant to us?)

Reply:
- https://toxcov.onrender.com/
- https://doxygen.onrender.com/
- https://webtox.onrender.com/
- Also automerge runs on render.com

Ok, I'll do that, but it'll take some time for me to find motivation for it, so I'll move this PR to the next milestone.